VERSION

```diff
@@ -1 +1,2 @@
-3.5.0
\ No newline at end of file
+3.6.0
+
```

Comment:
Nit: trailing empty line

```suggestion
```

Reply:
@gkurz The new line was removed in the last version bump. 

POSIX: 
> 3.206 Line
>A sequence of zero or more non- <newline> characters plus a terminating <newline> character.

We should have a new-line at EOF for every file.

Reply:
I fully agree but this PR is adding two new-line characters :wink:

```
$ od -c VERSION 
0000000   3   .   6   .   0  \n  \n
0000007
```

Reply:
How hard can it be to bump a VERSION? Zvonko: Yes!

Reply:
```
❯ hexdump -c VERSION 
0000000   3   .   6   .   0  \n                                        
0000006
```

Reply:
> How hard can it be to bump a VERSION? Zvonko: Yes!

I volunteered myself (as you would expect @gkurz ) to do the 3.7.0 release and I start to regret!